tools/matRad_compareDose.m

```diff
@@ -0,0 +1,290 @@
+%                  [1 0 0]: evaluate only basic plots
+%                  [0 1 0]: evaluate only line profiles
+%                  [0 0 1]: evaluate only DVH
+%   contours       (optional) specify if contours are plotted,
+%                  'on' or 'off'
+%   pln            (optional) specify BioModel for DVH plot
+%   criteria:      [1x2] vector (optional) specifying the distance to agreement
+%                  criterion; first element is percentage difference,
+%                  second element is distance [mm], default [3 3]
+%   n:             number of interpolations (optional). there will be 2^n-1
+%                  interpolation points. The maximum suggested value is 3.
+%                  default n=0
+%   localglobal:   parameter to choose between 'global' and 'local'
+%                  normalization (optional)
+%
+%
+% output
+%
+%   gammaCube:      result of gamma index calculation
+%   gammaPassRate:  rate of voxels passing the specified gamma criterion
+%                   evaluated for every structure listed in 'cst'.
+%                   note that only voxels exceeding the dose threshold are
+%                   considered.
+%   hfig:           Figure handle struct for all 3 figures and subplots,
+%                   indexed by plane names
+%
+% References gamma analysis:
+%   [1]  http://www.ncbi.nlm.nih.gov/pubmed/9608475
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2015 the matRad development team.
+%
+% This file is part of the matRad project. It is subject to the license
+% terms in the LICENSE file found in the top-level directory of this
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part
+% of the matRad project, including this file, may be copied, modified,
+% propagated, or distributed except according to the terms contained in the
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% check if cubes consistent
+if ~isequal(size(cube1),size(cube2))
+    error('dose cubes must be the same size\n');
+end
+
+if ~exist('localglobal','var')
+    localglobal = 'global';
+end
+if ~exist('n','var')
+    n = 0;
+end
+if ~exist('criteria','var')
+    criteria = [3 3];
+end
+if ~exist('cst','var') || isempty(cst)
+    cst = [];
+    skip = 1;
+    fprintf('[\bWarning: cst not specified, skipping DVH, isocenter in the center of the doseCube]\b\n');
+else
+    skip = 0;
+end
+if ~exist('pln','var') || isempty(pln)
+    pln = [];
+end
+if ~exist('contours','var') || isempty(contours)
+    contours = true;
+end
+if exist('contours','var') || ~isempty(contours)
+    if strcmp(contours,'on')
+        contours = true;
+    else
+        contours = false;
+    end
+end
+
+if (~exist('enable','var') || isempty(enable)) || strcmp(enable,'all')
+    enable = [1 1 1];
+end
+if enable(1)==0
+    gammaCube=[];
+    gammaPassRate=[];
+end
+%% Load colormap for difference
+load('diffCMap.mat');
```

Comment:
This should follow the standard matRad colormap worklfow. Add an diffCMap.m file to the colormaps folder, such that it get's recognized when using the matRad_getColormap function, and use it here.